app/models/policy.rb

```diff
@@ -38,13 +38,23 @@ def self.linkables
     Rails.cache.fetch('policy.linkables', expires_in: 5.minutes) do
       publishing_api.get_linkables(document_type: "policy").to_a
     end
+  rescue GdsApi::TimedOutException, GdsApi::HTTPServerError => e
+    Airbrake.notify_or_ignore(e)
+    # This call normally takes ~20ms. If it takes longer than a second, fetch a stale value from
+    # cache. Rails adds 5 minutes to the `expires_in` value above to generate the TTL it sends to
+    # memcached, so we will have 5 minutes of staleness before the key is evicted.
+    # If there's no key in the cache, just return an empty array. Incomplete data is more desirable
+    # than 504s / 500s
+    Rails.cache.fetch('policy.linkables') || []
```

Comment:
Are we sure about the empty array? This method provides the data for the `find` methods, which may start to wrongly return `nil` with this in place. Don't know what effect that could have.